dockerfile

```diff
@@ -0,0 +1,36 @@
+FROM ubuntu:18.04  
+
+# Installing  packages for the build
+RUN apt-get -y update  && apt-get install -y \
+build-essential \
+libboost-all-dev \
+libtiff5-dev \
+wget \
+make
+
+ENV LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/home/fftw/lib:/usr/lib/x86_64-linux-gnu
+ 
+WORKDIR /home/fftw
+RUN wget http://www.fftw.org/fftw-3.3.8.tar.gz
+RUN tar xfz fftw-3.3.8.tar.gz
+RUN rm fftw-3.3.8.tar.gz
+WORKDIR /home/fftw/fftw-3.3.8/
+RUN ./configure --prefix=/home/fftw
+RUN make install
+
+COPY . /home/
+WORKDIR /home/
+ENV LDFLAGS="-L/home/fftw/lib -L/usr/lib/x86_64-linux-gnu"
+ENV CPPFLAGS="-I/home/fftw/include"
+RUN ./configure --prefix=/home/ LIBS='-lboost_iostreams  -lboost_system -lboost_filesystem' CXXFLAGS='-g -O2 -fopenmp'
```

Comment:
what is boost needed for?  I thought that was for ome

Reply:
I removed boost. boost will be needed in the next PR for validating directory paths.